### ISSUE TITLE
How to use "resnet50_imagenet.pth"

### ISSUE THREAD

**Issue by ybpaopao** [multimodallearning/pytorch-mask-rcnn#72]
I want to use the pre-trained weighted "resnet50_imagenet.pth". How to use it? Can I just change some parameters or have to write my own codes in MaskRcnn class?
